senior_teacher.rb

```diff
@@ -50,4 +34,23 @@ def set_performance_rating(rating)
   def lead_training_session
     puts "Hey newbie!  Here are some common pitfalls.  Don't fall in them!"
   end
```

Comment:
set_performance_rating is still common between the ApprenticeTeacher and SeniorTeacher classes. How will you extract it ?